### DISCUSSION TITLE
Radiocomando Allarme

### DISCUSSION THREAD

**Discussion by Saint72** [indomus/forum#744] [GENERALE - Discussioni generiche]
Buongiorno a tutti.
Mi sto approcciando da poco al mondo domotico ed in previsione dei lavori di ristrutturazione di casa mia, sto studiando un bel po' sul vostro interessantissimo sito.
Ho un'allarme con telecomando multifunzione a 433 MHz, come faccio a sapere se è un Rolling o se i comandi sono statici?
Grazie
Marco

**Comment by KC-inDomus** [comment]
Prova ad usare, con le debite differenze, [questa guida](https://indomus.it/progetti/integrare-un-sistema-dallarme-etiger-s5-su-home-assistant-via-radiofrequenza-e-mqtt/). Se acquisendo i codici vedi che cambiano, allora sei in presenza di un rolling-code (e allora puoi usare [questa alternativa](https://indomus.it/progetti/domotizzare-un-sistema-dallarme-tradizionale-con-mqtt-e-node-red-parte-1/)).

**Reply by Saint72** [reply]
Grazie per il consiglio. Il sonoff rf bridge di cui si parla nella guida immagino che funzioni pure per gli apricancello FAAC tm433. Così con un unica spesa ho una doppia resa.
Grazie di nuovo
